infrastructure/app-insights.tf

```diff
@@ -18,5 +18,5 @@ resource "azurerm_application_insights" "appinsights" {
 resource "azurerm_key_vault_secret" "appinsights-connection-string" {
   name         = "civil-sdt-appinsights-connection-string"
   value        = azurerm_application_insights.appinsights.connection_string
-  key_vault_id = module.civil_sdt_key_vault.key_vault_id
+  key_vault_id = data.azurerm_key_vault.key_vault.id
```

Comment:
Change key_vault_id value to `module.civil_sdt_key_vault.key_vault_id` as secret needs to be created in the civil-sdt key vault.